dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/controller/ExecutorController.java

```diff
@@ -163,9 +169,18 @@ public Result startProcessInstance(@Parameter(hidden = true) @RequestAttribute(v
         if (timeout == null) {
             timeout = Constants.MAX_TASK_TIMEOUT;
         }
-        Map<String, String> startParamMap = null;
+        List<Property> startParamList = null;
         if (startParams != null) {
-            startParamMap = JSONUtils.toMap(startParams);
+            JsonElement jsonElement = JsonParser.parseString(startParams);
+            boolean isJson = jsonElement.isJsonObject();
+            if (isJson) {
+                Map<String, String> startParamMap = JSONUtils.toMap(startParams);
+                startParamList = startParamMap.entrySet().stream()
+                        .map(entry -> new Property(entry.getKey(), Direct.IN, DataType.VARCHAR, entry.getValue()))
+                        .collect(Collectors.toList());
+            } else {
+                startParamList = JSONUtils.toList(startParams, Property.class);
+            }
         }
```

Comment:
Add a utils to transform the json to Propertity list

Reply:
> Add a utils to transform the json to Propertity list

done